devkit/README.md

```diff
@@ -6,7 +6,7 @@ been changed.
 
 ## Getting Started
 
-You'll need [Node.js](https://nodejs.org/), specifically the active LTS version or higher.
+You'll need [Node.js](https://nodejs.org/), specifically LTS 22 or higher.
```

Comment:
I don't think you have to change this. The previous wording "the active LTS" version is currently 22 and less likely to go out of date.

Node 24 will be the active LTS version in about 1 month and a bit. 

https://endoflife.date/nodejs

Reply:
We'll forget to update the docs when we switch to the next version.